examples/plot_simulate_alpha.py

```diff
@@ -42,24 +48,71 @@
 # simulation. Each burst consists of a pair (2) of spikes, spaced 10 ms apart.
 # The occurrence of each burst is jittered by a random, normally distributed
 # amount (20 ms standard deviation). We repeat the burst train 10 times, each
-# time with unique randomization. The drive is only connected to the distal
+# time with unique randomization. The drive is only connected to the proximal
 # (dendritic) AMPA synapses on L2/3 and L5 pyramidal neurons.
 net = Network(params)
 
-weights_ampa = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+location = 'proximal'
+burst_std = 20
+weights_ampa_p = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+syn_delays_p = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                'L5_basket': 1., 'L5_pyramidal': 1.}
+
 net.add_bursty_drive(
-    'bursty', tstart=50., burst_rate=10, burst_std=20., numspikes=2,
-    spike_isi=10, repeats=10, location='distal', weights_ampa=weights_ampa,
-    seedcore=4)
+    'alpha_prox', tstart=50., burst_rate=10, burst_std=burst_std, numspikes=2,
+    spike_isi=10, repeats=10, location=location, weights_ampa=weights_ampa_p,
+    synaptic_delays=syn_delays_p, seedcore=13)
 
 dpl = simulate_dipole(net)
 
 trial_idx = 0  # single trial simulated
 dpl[trial_idx].plot()
 
 ###############################################################################
-# We can confirm that what we simulate is indeed 10 Hz activity.
+# We can confirm that what we simulate is indeed 10 Hz activity by plotting the
+# power spectral density. Note that the SciPy-function
+# `~scipy.signal.spectrogram` is used to create the plot.
 import matplotlib.pyplot as plt
 from hnn_core.viz import plot_spectrogram
 tmin = 20  # exclude initial burn-in period
 plot_spectrogram(dpl[trial_idx], fmin=0., fmax=40., tmin=tmin)
+
+###############################################################################
+# The next step is to add a simultaneous 10 Hz distal drive. Due to the
+# stochasticity of input spike timing, the proximal and distal spikes
+# occasionally arrive at the same time which will result in a beta frequency
+# (15-30 Hz) event.
+location = 'distal'
+burst_std = 20
+weights_ampa_d = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+syn_delays_d = {'L2_basket': 5., 'L2_pyramidal': 5.,
+                'L5_basket': 5., 'L5_pyramidal': 5.}
```

Comment:
```suggestion
syn_delays_d = {'L2_pyramidal': 5., 'L5_pyramidal': 5.}
```

Reply:
Are these default values for basket cells or something? In any case I am getting this error when I try it:
**ValueError: synaptic_delays is either a common float or needs to be specified as a dict for each cell type**

Reply:
Nope: the delays need to be specified for any cell type for which weights are given. Since you're only connecting to pyramidal cells, there's no reason to include basket delays in the dict.

This works for me on this branch, could you try again before we investigate further?